src/components/Detail/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { func, object } from 'prop-types'
+
+import { Header } from '../'
+import GifItem from '../../containers/GifItem'
+import { DetailWrapper, GifWrapper, TitleWrapper } from './Detail.style'
+
+class Detail extends React.Component {
+  componentWillMount() {
+    this.props.initialSetup(this.props.match.params.id)
+  }
+
+  render() {
+    const { giph } = this.props
+
+    return (
+      <div>
+        <Header />
+        <DetailWrapper>
+        { giph !== null &&
+          <div>
+            <GifWrapper>
+              <GifItem imageUrl={giph.images.original.url} alt={giph.title} isDetail={true} id={giph.id}/>
```

Comment:
you can define truthy boolean props like `<GiftItem isDetail />` property without =true